### DISCUSSION TITLE
[HELP] Cómo puedo hacer que una fecha sea superior a otra en un survey?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#203] [2. Questions and Answers] [answered]
Hola no sé si está bien formulario la pregunta pero dejo la descripción

Tengo dos fechas, una fecha inicio y una fecha fin y necesito que la fecha fin sea posterior a la fecha inicio.

Ejemplo

Fecha inicio: 24-05-2021
Fecha Fin: >= 24-05-2021, pero no < 24-05-2021.

Espero que me puedan ayudar

Gracias :>

**Comment by seiturra** [comment, marked as answer]
@NicolasPLG 

Con question exec validate se puede solucionar esto:

*  __Question1__: resetea a Question2
*  __Question2__: valida que Question1 sea menor que si misma.

Ejemplo exec.validate:

# Contexto
```
responses#identifier_question_1,responses#self
```
# Código
```
function run() {
  var initTime = new Date(context['responses#identifier_question_1']);
  var endTime = new Date(context['responses#self']);
  try {
    if (initTime > endTime) { 
      return [{
        cmd: 'RESULT',
        result: false,
        value: 'Inicio tiempo muerto ('+ context['responses#identifier_question_1'] + ') es mayor que Fin tiempo muerto ('+ context['responses#self'] + ')'
      }];
    } else {
      return [{
        cmd: 'RESULT',
        result: true
      }];
    }
  } catch (e) {
    return [{
      cmd: 'RESULT',
      result: false,
      value: ' Ha ocurrido un error inesperado.'.concat(e, ').')
    }];
  }
};
```

**Reply by NicolasPLG** [reply]
Hola @SeIturra gracias por la información, lo probaré y te comento.

**Comment by NicolasPLG** [comment]
@SeIturra Funciona! :D, cambié la frase algo más breve

![image](https://user-images.githubusercontent.com/61419471/119401183-261f1c00-bca9-11eb-9dea-35adc24c1ec2.png)


SAlu2

**Reply by acedward** [reply]
y eso se ve super mal?!

**Reply by NicolasPLG** [reply]
la forma del cuadro?, como un bug o feature @acedward ?
